42_grml

```diff
@@ -74,10 +74,18 @@ get_dependencies() {
     break
   done
   echo $dependencies
-
 }
 
-
+get_iso_identifier()
+{
+  local iso=${1}
```

Comment:
"local" sadly isn't strictly POSIX, but AFAICT it shouldn't be a problem for bash, dash,... or any relevant shells that might be present for /bin/sh on Debian or alike systems.

But please double quote the "$1" (and nitpick: I prefer to use $X for single char variables and ${foo} for multi char variables)

Reply:
(FTR, the `$..$` (without single backticks) in the comment above are rendered as mathematical expression, see: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions. I just fixed it with `<span>$</span>` without putting them in single backticks.)